src/includes/classes/MenuPageOptions.php

```diff
@@ -1171,8 +1169,51 @@ public function __construct()
             echo '   </div>'."\n";
             echo '</div>'."\n";
         }
+        /* ----------------------------------------------------------------------------------------- */
+
+        if (IS_PRO || $this->plugin->isProPreview()) {
+            echo '<div class="plugin-menu-page-panel'.(!IS_PRO ? ' pro-preview' : '').'">'."\n";
+
+            echo '   <a href="#" class="plugin-menu-page-panel-heading" data-pro-version-only="'.(!IS_PRO ? __('pro version only', SLUG_TD) : '').'">'."\n";
+            echo '      <i class="si si-tablet"></i> '.__('Mobile-Adaptive Mode', SLUG_TD)."\n";
+            echo '   </a>'."\n";
+
+            echo '   <div class="plugin-menu-page-panel-body clearfix">'."\n";
+            echo '      <h3>'.__('<i class="si si-tablet"></i> <i class="si si-mobile"></i> Enable Mobile-Adaptive Mode?', SLUG_TD).'</h3>'."\n";
+            echo '      <p>'.__('<em><strong>Tip:</strong> Generally speaking, you should only enable this if your WordPress theme uses an \'Adaptive\' design, as opposed to a design that\'s \'Responsive\'—the way most WordPress themes are built.', SLUG_TD).'</em></p>'."\n";
+            echo '      <p><select name="'.esc_attr(GLOBAL_NS).'[saveOptions][mobile_adaptive_salt_enable]" data-target=".-mobile-adaptive-options">'."\n";
+            echo '          <option value="0"'.(!IS_PRO ? '' : selected($this->plugin->options['mobile_adaptive_salt_enable'], '0', false)).'>'.__('No, my theme is Responsive; i.e., not Adaptive.', SLUG_TD).'</option>'."\n";
+            echo '          <option value="1"'.(!IS_PRO ? ' selected' : selected($this->plugin->options['mobile_adaptive_salt_enable'], '1', false)).'>'.__('Yes, create multiple cache variations based on mobile device type.', SLUG_TD).'</option>'."\n";
+            echo '      </select></p>'."\n";
+
+            if (!version_compare(PHP_VERSION, '5.6', '>=')) {
+                echo '<p class="warning">'.sprintf(__('<strong>PROBLEM:</strong> This feature requires PHP 5.6 (or higher). You\'re currently running PHP v%1$s.', SLUG_TD), esc_html(PHP_VERSION)).'</p>'."\n";
```

Comment:
This needs the following changes: 

- Should be `class="error"`
- The prefix should say "PHP Version:" instead of "PROBLEM:"
- "PHP 5.6" needs a `v`: "PHP v5.6"
- An additional note should be added to indicate where to get help. "Please contact your web hosting company for assistance."

![2016-12-12_21-49-52](https://cloud.githubusercontent.com/assets/53005/21125639/2e382fde-c0b5-11e6-9f6a-4340241e6143.png)